tests/entities_column_value_simple_tests.py

```diff
@@ -700,13 +699,12 @@ def test_should_set_a_status_column_value_with_valid_integer_index_value():
     eq_(column_value.value,'Done')
 
 
-def test_should_email_column_value_with_no_api_input_data():
```

Comment:
It looks like when you merged v2.0 into your branch, you overwrote some of the existing unit tests.

Please revert this change and fix it such that we can keep all of our unit tests.